docker/swarm/docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
         gid: "999"
         mode: 0440
     env_file: scim.env
+    depends_on: [ 'redis' ]
```

Comment:
Is this something we intended to include in this version bump PR? Seems reasonable, but I thought I would double check.

Reply:
I thought I'd include it as it's a minor fix, and I didn't want to create a full PR for a one-line change. It can be broken out if needed, though.